Made the game responsive

DNA5769 opened the pull request:
Fixes #29

DNA5769 committed:
Prompting user to switch to landscape mode if not already

DNA5769 committed:
Made it responsive?

DNA5769 committed:
- I should probably study for OR exam
- Added some documentation

DNA5769 commented:
Can you review this @dev-lovedeep

dev-lovedeep commented on index.js:
```diff
@@ -10,10 +14,13 @@ function toggleSound(){
 // Wrapping entire code in anonymous function and calling it, so that user doesn't have access to cardImageSrcs
 (() => {
     let level = window.prompt("Choose a level", "1/2/3");
+    if (level !== '1' && level !== '2' && level !== '3')
+        level = '1'; // Setting default level to 1
```
bonus for this :balloon:

dev-lovedeep commented on index.js:
```diff
@@ -1,3 +1,7 @@
+// Suggesting user to switch to landscape mode if their currently in portrait mode
+if (screen.availHeight > screen.availWidth)
+    alert("Please switch to landscape mode for a better experience!");
```
this will check only one, instead add an event listener

dev-lovedeep reviewed (approved):
make this small change, and then we can merge

DNA5769 committed:
Removing orientation check from JS

DNA5769 committed:
Setting up game such that user can play game only if screen is in
landscape orientation
(Hmmm I wonder any other awesome website developed by awesome people
which only allow us to play the game if the screen is a particular size)

DNA5769 commented:
Adding a event listener for resize and putting alerts in it, made the game very buggy.
So instead I decided to display the message through text in the website.

dev-lovedeep commented on style.css:
```diff
@@ -203,3 +206,39 @@ header{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    font-size: 8rem;
```
this is way too big change it to something like 2 or 3rem

DNA5769 committed:
Made size of orientation message to be smaller
